net/haproxy/Portfile

```diff
@@ -52,6 +56,8 @@ build.target        TARGET=osx
 pre-build {
     build.args      LD="${configure.cc}"\
                     LDFLAGS="${configure.ldflags}" \
+                    SSL_INC=${prefix}/include/openssl-1.1 \
+                    SSL_LIB=${prefix}/lib/openssl-1.1 \
```

Comment:
The `openssl` portgroup provides some convenience variables for use here.

First off is `[openssl::install_area]`.  This is the base directory for whichever OpenSSL is currently being provided by the portgroup.

Then you have `[openssl::include_dir]` and `[openssl::lib_dir]`, which both build off of `[openssl::install_area]`. You would use both of these for your `SSL_INC` and `SSL_LIB` args, respectively:

```
    build.args      LD="${configure.cc}"\
                    LDFLAGS="${configure.ldflags}" \
                    SSL_INC=[openssl::include_dir] \
                    SSL_LIB=[openssl::lib_dir] \
                    ...
```